src/entry.ts

```diff
@@ -129,10 +129,10 @@ function processGenerationOption(session: Session<CodeModel>) {
     let isSdkNeeded = !cliCore;
     const generateSdk = AzConfiguration.getValue(CodeGenConstants.generateSDK);
     isSdkNeeded = isNullOrUndefined(generateSdk) ? isSdkNeeded : generateSdk === GenerateSdk.Yes;
-    const compatibleLevel =
-        AzConfiguration.getValue(CodeGenConstants.compatibleLevel) || cliCore
-            ? CompatibleLevel.Track1
-            : CompatibleLevel.Track2;
+    let compatibleLevel = cliCore ? CompatibleLevel.Track1 : CompatibleLevel.Track2;
```

Comment:
Could you rename it to compatibilityLevel?

Reply:
Why? Just think it may cause confusion for our customer who already had tried --compatible-level before.